src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-comparison.mdx

```diff
@@ -0,0 +1,1139 @@
+  </Collapser>
+</CollapserGroup>
+
+## Serverless [#serverless]
+
+New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry:  🟡 Limited support
+
+OpenTelemetry serverless function telemetry can be reported, stored, and visualized with custom dashboards. A dedicated view of this telemetry is not yet available.
+
+## Alert [#alerting]
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="alerts-collap"
+    title="Alerts"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ✅ 
+
+    You can configure alerts on OpenTelemetry data using conditions written in NRQL. Click any chart’s “...” menu to get started.
+
+    Conditions, violations, incidents, and issues all work for any entity.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="recommended-conditions"
+    title="Recommended alert conditions"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ❌
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="alert-condit-list"
+    title="Alert conditions list UI"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ✅ 
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="alert-viol-list"
+    title="Alert violations list UI"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ❌
+  </Collapser>
+</CollapserGroup>
+
+## New Relic platform
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="logs-context"
+    title="Logs in context"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ✅ 
+
+    For OpenTelemetry services, just ensure that the `service.name` attribute is present in logs to associate them with a service entity.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="slm"
+    title="Service Level Management (SLM) (Beta)"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ✅ 
+
+    OpenTelemetry service entities do not have a dedicated SLM UI, but SLIs (service level indicators) can be viewed for them in a workload or in the mini-overview. SLIs for OpenTelemetry services can only be created from log, span, or span event data.
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="quick-recipes"
+    title="Quick-setup recipes"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ✅ 
+
+    For New Relic agents, click [Add data](https://one.newrelic.com/marketplace?state=7ca7c800-845d-8b31-4677-d21bcc061961) in the UI.
+
+    For OpenTelemetry, you can start with our [setup instructions](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-setup/), which guides to various sites to complete the installation.
+  </Collapser>
+
+  <Collapser
+    className="freq-link"
+    id="exemplars"
+    title="Exemplars"
+  >
+    New Relic agents: ✅ &nbsp; &nbsp; &nbsp; OpenTelemetry: ✅ 
```

Comment:
`limited` for NR agents, `no` for OTel, since we can't ingest them for all customers yet and have no UI.